vendor/engines/split_accounts/spec/engine_helper.rb

```diff
@@ -0,0 +1,9 @@
+# Rspec configurations for this engine
+RSpec.configure do |config|
+  
+  # Skip split_accounts tests if split_accounts feature is disabled
+  unless SettingsHelper.feature_on?(:split_accounts)
```

Comment:
I wasn't a fan of skipping the tests as well.  I was thinking maybe we could include all split_account code and only use the `split_accounts` feature to update `AccountConfig.new.account_types`.  But then we're loading split accounts code even if we're not using it.